### PR TITLE
Fixed formatting

### PR DESCRIPTION
**Pull request by dshevtsov**
## This PR is a:

- Content fix or rewrite

## Summary

When this pull request is merged, it will convert HTML links and enable markdown in notes.

### CODE REVIEW

**Comment by dshevtsov** [comment]
@mhauri for sure
but it requires more complex find/replace logic and more time

File: guides/v2.0/architecture/archi_perspectives/components/modules/mod_depend.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ Although Magento architecture favors loosely coupled software components, module
 
 * libraries (either Magento Framework {% glossarytooltip 08968dbb-2eeb-45c7-ae95-ffca228a7575 %}library{% endglossarytooltip %} or third party libraries)
 
-<div class="bs-callout bs-callout-warning" id="warning">
+<div class="bs-callout bs-callout-warning" markdown="1">
```
Comment (shrielenee) [inline guides/v2.0/architecture/archi_perspectives/components/modules/mod_depend.md]: This is called out as a warning, but in the actual content it says "Note:". This seems incorrect.

File: guides/v2.0/architecture/archi_perspectives/components/modules/mod_depend_types.md [hunk 2]
```diff
@@ -35,7 +35,7 @@ Modules with a *soft dependency* on another module can function properly without
 
 * The module extends another module's {% glossarytooltip 73ab5daa-5857-4039-97df-11269b626134 %}layout{% endglossarytooltip %}.
 
-<div class="bs-callout bs-callout-warning" id="warning">
+<div class="bs-callout bs-callout-warning" markdown="1">
```
Comment (shrielenee) [inline guides/v2.0/architecture/archi_perspectives/components/modules/mod_depend_types.md]: This is called out as a warning, but in the actual content it says "Note:". This seems incorrect.

File: guides/v2.0/comp-mgr/trouble/cman/tshoot_backup.md [hunk 3]
```diff
@@ -11,28 +11,28 @@ functional_areas:
...
+*	[Backup fails](#trouble-backup-all)
 
 ### Insufficient disk space   {#trouble-backup-space}
 
 If the backup failed because of insufficient disk space, you should typically free up disk space by moving some files to another storage device or drive. However, there might be other ways to resolve the issue. See one of the following resources for tips:
 
-*	<a href="http://www.cyberciti.biz/datacenter/linux-unix-bsd-osx-cannot-write-to-hard-disk" target="_blank">8 Tips to Solve Linux & Unix Systems Hard Disk Problems Like Disk Full Or Can’t Write to the Disk</a>
-*	<a href="http://serverfault.com/questions/315181/df-says-disk-is-full-but-it-is-not" target="_blank">serverfault: df says disk is full, but it is not</a>
-*	<a href="http://unix.stackexchange.com/questions/125429/tracking-down-where-disk-space-has-gone-on-linux" target="_blank">unix.stackexchange.com: Tracking down where disk space has gone on Linux? </a>
+*	[8 Tips to Solve Linux & Unix Systems Hard Disk Problems Like Disk Full Or Can’t Write to the Disk](http://www.cyberciti.biz/datacenter/linux-unix-bsd-osx-cannot-write-to-hard-disk){: target="_blank"}
```
Comment (shrielenee) [inline guides/v2.0/comp-mgr/trouble/cman/tshoot_backup.md]: I don't think there is supposed to be a space between `:` and `target` here.

File: guides/v2.0/comp-mgr/trouble/cman/tshoot_backup.md [hunk 4]
```diff
@@ -11,28 +11,28 @@ functional_areas:
...
+*	[serverfault: df says disk is full, but it is not](http://serverfault.com/questions/315181/df-says-disk-is-full-but-it-is-not){: target="_blank"}
```
Comment (shrielenee) [inline guides/v2.0/comp-mgr/trouble/cman/tshoot_backup.md]: I don't think there is supposed to be a space between `:` and `target` here.

File: guides/v2.0/comp-mgr/trouble/cman/tshoot_backup.md [hunk 5]
```diff
@@ -11,28 +11,28 @@ functional_areas:
...
+*	[unix.stackexchange.com: Tracking down where disk space has gone on Linux? ](http://unix.stackexchange.com/questions/125429/tracking-down-where-disk-space-has-gone-on-linux){: target="_blank"}
```
Comment (shrielenee) [inline guides/v2.0/comp-mgr/trouble/cman/tshoot_backup.md]: I don't think there is supposed to be a space between `:` and `target` here.

File: guides/v2.0/comp-mgr/trouble/cman/tshoot_backup.md [hunk 6]
```diff
@@ -11,28 +11,28 @@ functional_areas:
...
 
 ### Operating system error   {#trouble-backup-os}
 
 Unfortunately, we can't recommend anything specific because of the variety of errors you might encounter. We can suggest, however, you:
 
 *	Contact your system administrator
-*	Search public forums like <a href="http://unix.stackexchange.com" target="_blank">stackexchange</a> or <a href="http://stackoverflow.com" target="_blank">stackoverflow</a>
-*	Open a <a href="https://github.com/magento/magento2/issues" target="_blank">GitHub issue</a> and we'll try to help
+*	Search public forums like [stackexchange](http://unix.stackexchange.com){: target="_blank"} or [stackoverflow](http://stackoverflow.com){: target="_blank"}
```
Comment (shrielenee) [inline guides/v2.0/comp-mgr/trouble/cman/tshoot_backup.md]: Should be `StackExchange`, and I don't think there is supposed to be a space between `:` and `target` here.

Comment (shrielenee) [reply guides/v2.0/comp-mgr/trouble/cman/tshoot_backup.md]: Also should be `Stack Overflow`

File: guides/v2.0/comp-mgr/trouble/cman/tshoot_backup.md [hunk 7]
```diff
@@ -11,28 +11,28 @@ functional_areas:
...
+*	Open a [GitHub issue](https://github.com/magento/magento2/issues){: target="_blank"} and we'll try to help
```
Comment (shrielenee) [inline guides/v2.0/comp-mgr/trouble/cman/tshoot_backup.md]: I don't think there is supposed to be a space between `:` and `target` here.

File: guides/v2.0/comp-mgr/trouble/cman/were-sorry.md [hunk 8]
```diff
@@ -25,21 +25,21 @@ You might not have entered your authentication keys in the {% glossarytooltip 18
...
 
-If that doesn't work, try generating <a href="{{ page.baseurl }}/install-gde/prereq/connect-auth.html">new authentication keys</a> and enter those in the Admin. Then try your upgrade again.
+If that doesn't work, try generating [new authentication keys]({{ page.baseurl }}/install-gde/prereq/connect-auth.html) and enter those in the Admin. Then try your upgrade again.
 
 ### Problem: the updater application isn't initialized {#updater}
 
-In some cases (especially if you downloaded the Magento software from <a href="https://packagist.org/" target="_blank">packagist</a>), the updater application might not be initialized. (A common way for this to happen is to not specify our `https://repo.magento.com` repository in the `composer create-project` command.)
+In some cases (especially if you downloaded the Magento software from [packagist](https://packagist.org/){: target="_blank"}), the updater application might not be initialized. (A common way for this to happen is to not specify our `https://repo.magento.com` repository in the `composer create-project` command.)
```
Comment (shrielenee) [inline guides/v2.0/comp-mgr/trouble/cman/were-sorry.md]: I don't think there is supposed to be a space between `:` and `target` here.

File: guides/v2.0/comp-mgr/upgrader/ce-ee-upgrade-start.md [hunk 9]
```diff
@@ -13,21 +13,21 @@ functional_areas:
...
 	<p>You must be authorized for {{site.data.var.ee}} to perform the tasks discussed in this topic.</p>
 </div>
 
 ## Prerequisites   {#compman-prereq}
 
-Before continuing, complete all tasks discussed in <a href="{{ page.baseurl }}/comp-mgr/prereq/prereq_compman.html">Prerequisites</a>.
+Before continuing, complete all tasks discussed in [Prerequisites]({{ page.baseurl }}/comp-mgr/prereq/prereq_compman.html).
 
-In addition, you might need to install the {% glossarytooltip bf703ab1-ca4b-48f9-b2b7-16a81fd46e02 %}PHP{% endglossarytooltip %} <a href="http://php.net/manual/en/book.bc.php" target="_blank">`bcmath`</a> extension, which is required by {{site.data.var.ee}}. Examples follow:
+In addition, you might need to install the {% glossarytooltip bf703ab1-ca4b-48f9-b2b7-16a81fd46e02 %}PHP{% endglossarytooltip %} [`bcmath`](http://php.net/manual/en/book.bc.php){: target="_blank"} extension, which is required by {{site.data.var.ee}}. Examples follow:
```
Comment (shrielenee) [inline guides/v2.0/comp-mgr/upgrader/ce-ee-upgrade-start.md]: I don't think there is supposed to be a space between `:` and `target` here.

File: guides/v2.0/comp-mgr/upgrader/ce-ee-upgrade-start.md [hunk 10]
```diff
@@ -13,21 +13,21 @@ functional_areas:
...
 
 *	CentOS (using the `webtatic` repository): `yum -y install php56w-bcmath`
 *	Ubuntu (using the `ppa:ondrej/php5-5.6` repository): `apt-get -y install php5-bcmath`
 
-<div class="bs-callout bs-callout-info" id="info">
-	<p>Make sure you are authorized for {{site.data.var.ee}} access before you continue. Contact <a href="http://support.magentocommerce.com" target="_blank">Magento Support</a> if you have questions.</p>
+<div class="bs-callout bs-callout-info" markdown="1">
```
Comment (shrielenee) [inline guides/v2.0/comp-mgr/upgrader/ce-ee-upgrade-start.md]: I don't think there is supposed to be a space between `:` and `target` here.

File: guides/v2.0/config-guide/bootstrap/mage-dirs.md [hunk 11]
```diff
@@ -18,11 +18,11 @@ The `MAGE_DIRS` environment variable enables you to specify custom base director
 
 ## Set MAGE\_DIRS   {#dirs-set}
 
-Specify an associative array where keys are constants from <a href="{{ site.mage2000url }}lib/internal/Magento/Framework/App/Filesystem/DirectoryList.php" target="_blank">\Magento\App\Filesystem\DirectoryList</a> and values are absolute paths of directories or their {% glossarytooltip a05c59d3-77b9-47d0-92a1-2cbffe3f8622 %}URL{% endglossarytooltip %} paths, respectively.
+Specify an associative array where keys are constants from [\\Magento\\App\\Filesystem\\DirectoryList]({{ site.mage2000url }}lib/internal/Magento/Framework/App/Filesystem/DirectoryList.php){: target="_blank"} and values are absolute paths of directories or their {% glossarytooltip a05c59d3-77b9-47d0-92a1-2cbffe3f8622 %}URL{% endglossarytooltip %} paths, respectively.
```
Comment (shrielenee) [inline guides/v2.0/config-guide/bootstrap/mage-dirs.md]: I don't think there should be two `\` in this file path. I don't think there is supposed to be a space between `:` and `target` here.

File: guides/v2.0/config-guide/bootstrap/mage-profiler.md [hunk 12]
```diff
@@ -22,7 +22,7 @@ Magento profiling enables you to:
 
 	You should be particularly interested in the list of *unused dependencies*, which are objects that were created because they were requested in some constructor, but were never used (that is, none of their methods were called). As a result, processor time and memory spent to create these dependencies are wasted.
 
-Magento provides the base functionality in <a href="{{ site.mage2000url }}lib/internal/Magento/Framework/Profiler.php" target="_blank">Magento\Framework\Profiler</a>.
+Magento provides the base functionality in [Magento\\Framework\\Profiler]({{ site.mage2000url }}lib/internal/Magento/Framework/Profiler.php){: target="_blank"}.
```
Comment (shrielenee) [inline guides/v2.0/config-guide/bootstrap/mage-profiler.md]: Not supposed to be two `\` and I don't think there is supposed to be a space between `:` and `target` here.

File: guides/v2.0/config-guide/bootstrap/mage-profiler.md [hunk 13]
```diff
@@ -31,13 +31,13 @@ Magento provides the base functionality in <a href="{{ site.mage2000url }}lib/in
 
 	You can also use one of the following values to enable a specific profiler:
 
-    - `csvfile` which uses <a href="{{ site.mage2000url }}lib/internal/Magento/Framework/Profiler/Driver/Standard/Output/Csvfile.php" target="_blank">Magento\Framework\Profiler\Driver\Standard\Output\Csvfile</a>
-    - Any other value (except `2`), including an empty value, which uses <a href="{{ site.mage2000url }}lib/internal/Magento/Framework/Profiler/Driver/Standard/Output/Html.php" target="_blank">Magento\Framework\Profiler\Driver\Standard\Output\Html</a>
+    - `csvfile` which uses [Magento\\Framework\\Profiler\\Driver\\Standard\\Output\\Csvfile]({{ site.mage2000url }}lib/internal/Magento/Framework/Profiler/Driver/Standard/Output/Csvfile.php){: target="_blank"}
```
Comment (shrielenee) [inline guides/v2.0/config-guide/bootstrap/mage-profiler.md]: Not supposed to be two `\` and I don't think there is supposed to be a space between `:` and `target` here.

Comment (shrielenee) [reply guides/v2.0/config-guide/bootstrap/mage-profiler.md]: Same for below

File: guides/v2.0/config-guide/bootstrap/magento-bootstrap.md [hunk 14]
```diff
@@ -14,19 +14,19 @@ functional_areas:
 
 ## Overview of bootstrapping   {#config-boot-overview}
 
-To run the Magento application, the following actions are implemented in <a href="{{ site.mage2000url }}index.php" target="_blank">index.php</a>:
+To run the Magento application, the following actions are implemented in [index.php]({{ site.mage2000url }}index.php){: target="_blank"}:
```
Comment (shrielenee) [inline guides/v2.0/config-guide/bootstrap/magento-bootstrap.md]: I don't think there is supposed to be a space between `:` and `target` here.

File: guides/v2.0/config-guide/bootstrap/magento-bootstrap.md [hunk 15]
```diff
@@ -14,19 +14,19 @@ functional_areas:
...
 
-*	Include <a href="{{ site.mage2000url }}app/bootstrap.php" target="_blank">app/bootstrap.php</a> which performs essential initialization routines, such as error handling, initializing the autoloader, setting profiling options, setting the default timezone, and so on.
-*	Create an instance of <a href="{{ site.mage2000url }}lib/internal/Magento/Framework/App/Bootstrap.php" target="_blank">\Magento\Framework\App\Bootstrap</a>. <!-- It requires initialization parameters to be specified in constructor.  Normally, the $_SERVER super-global variable is supposed to be passed there. -->
-*	Create a Magento application instance (<a href="{{ site.mage2000url }}lib/internal/Magento/Framework/AppInterface.php" target="_blank">\Magento\Framework\AppInterface</a>).
+*	Include [app/bootstrap.php]({{ site.mage2000url }}app/bootstrap.php){: target="_blank"} which performs essential initialization routines, such as error handling, initializing the autoloader, setting profiling options, setting the default timezone, and so on.
```
Comment (shrielenee) [inline guides/v2.0/config-guide/bootstrap/magento-bootstrap.md]: I don't think there is supposed to be a space between `:` and `target` here.

File: guides/v2.0/config-guide/bootstrap/magento-bootstrap.md [hunk 16]
```diff
@@ -14,19 +14,19 @@ functional_areas:
...
+*	Create an instance of [\\Magento\\Framework\\App\\Bootstrap]({{ site.mage2000url }}lib/internal/Magento/Framework/App/Bootstrap.php){: target="_blank"}. <!-- It requires initialization parameters to be specified in constructor.  Normally, the $_SERVER super-global variable is supposed to be passed there. -->
```
Comment (shrielenee) [inline guides/v2.0/config-guide/bootstrap/magento-bootstrap.md]: I'm going to stop my review here. Please review all of the changes for double `\` and the errant space between `:` and `target` throughout.

Comment (shrielenee) [reply guides/v2.0/config-guide/bootstrap/magento-bootstrap.md]: I'll pick up my review when those changes have been made.

File: guides/v2.0/architecture/archi_perspectives/components/modules/mod_depend.md (hunk 1 above)
Comment (dshevtsov) [reply guides/v2.0/architecture/archi_perspectives/components/modules/mod_depend.md]: This issue is out of scope in this PR.

File: guides/v2.0/architecture/archi_perspectives/components/modules/mod_depend_types.md (hunk 2 above)
Comment (dshevtsov) [reply guides/v2.0/architecture/archi_perspectives/components/modules/mod_depend_types.md]: This issue is out of scope in this PR.

File: guides/v2.0/comp-mgr/trouble/cman/tshoot_backup.md (hunk 6 above)
Comment (dshevtsov) [reply guides/v2.0/comp-mgr/trouble/cman/tshoot_backup.md]: This issue is out of scope in this PR.

**Comment by dshevtsov** [comment]
@shrielenee the `\\` is `\` with escaping. See more details in the Kramdown documentation [Automatic and Manual Escaping]( https://kramdown.gettalong.org/syntax.html#automatic-and-manual-escaping).

The `target` attributes were introduced as an output of the original Kramdown converter (see the [tool used for the conversion](https://github.com/dshevtsov/html-to-kramdown/blob/9d4875a85ad3c429314f9b4868f81de68b845f00/lib/converter.rb#L10)). The [Kramdown syntax](https://kramdown.gettalong.org/syntax.html) allows both options `{:param}` and `{: param}`.

File: guides/v2.0/install-gde/trouble/tshoot_install-log.md [hunk 17]
```diff
@@ -22,13 +22,13 @@ Running Magento processes at the same time might result in problems creating the
...
+Review your setting for `open_basedir` in `php.ini`. The Setup Wizard uses the [sys\_get\_temp\_dir ( void )](http://php.net/manual/en/function.sys-get-temp-dir.php){: target="_blank"} PHP call to get the value of the temporary directory. If [open\_basedir](http://php.net/manual/en/ini.core.php#ini.open-basedir){: target="_blank"} is set to refuse connections to a directory specified by `sys_get_temp_dir`, the installation fails.
 
 To resolve the issue, change the value of `open_basedir` and restart the web server.
 
 If you're not sure how to change this value, use the following steps:
 
-1.	If you haven't already done so, create <a href="{{ page.baseurl }}/install-gde/prereq/optional.html#install-optional-phpinfo">phpinfo.php</a>.
+1.	If you haven't already done so, create [phpinfo.php]({{ page.baseurl }}/install-gde/prereq/optional.html#install-optional-phpinfo).
```
Comment (shrielenee) [inline guides/v2.0/install-gde/trouble/tshoot_install-log.md]: Add backticks to file paths for code formatting in this pge.

File: guides/v2.0/install-gde/trouble/tshoot_mysql_table-open-cache.md [hunk 18]
```diff
@@ -19,7 +19,7 @@ During installation, the following message displays:
 
 ### Solution
```
Comment (shrielenee) [inline guides/v2.0/install-gde/trouble/tshoot_mysql_table-open-cache.md]: Should table\_open\_cache be in code formatting?

File: guides/v2.0/migration/migration-tool-internal-spec.md [hunk 19]
```diff
@@ -111,7 +111,7 @@ Script that runs migration process is located at magento-root/bin/magento
 
 The Schema for configuration file `config.xsd` is placed under `etc/directory`. Default configuration file `config.xml.dist` is created for each version of Magento 1.x. It is placed in separate directories under `etc/`.
 
-Default configuration file can be replaced by custom one using CLI (see <a href="{{ page.baseurl }}/migration/migration-migrate.html">--config <code>&lt;value&gt;</code> parameter</a>).
+Default configuration file can be replaced by custom one using CLI (see [--config `<value>` parameter]({{ page.baseurl }}/migration/migration-migrate.html)).
```
Comment (shrielenee) [inline guides/v2.0/migration/migration-tool-internal-spec.md]: Add articles: "The default configuration file can be replaced by a custom one..."

File: guides/v2.0/mtf/mtf_entities.md [hunk 20]
```diff
@@ -4,36 +4,36 @@ title: Functional Testing Framework Entities
...
+  - [Create new fixture]({{ page.baseurl }}/mtf/mtf_entities/mtf_fixture.html#mtf_fixture_create)
+  - [Read and update new fixture]({{ page.baseurl }}/mtf/mtf_entities/mtf_fixture.html#mtf_fixture_read)
+  - [Add repository to the fixture field]({{ page.baseurl }}/mtf/mtf_entities/mtf_fixture.html#mtf_fixture_repositoy)
+  - [Add data source to the fixture field]({{ page.baseurl }}/mtf/mtf_entities/mtf_fixture.html#mtf_fixture_source)
+  - [Merge fixtures]({{ page.baseurl }}/mtf/mtf_entities/mtf_fixture.html#mtf_fixture_merge)
+  - [Extend fixture]({{ page.baseurl }}/mtf/mtf_entities/mtf_fixture.html#mtf_fixture_extend)
   
   
-- <a href="{{ page.baseurl }}/mtf/mtf_entities/mtf_fixture-repo.html">Fixture Repository</a>
+- [Fixture Repository]({{ page.baseurl }}/mtf/mtf_entities/mtf_fixture-repo.html)
```
Comment (shrielenee) [inline guides/v2.0/mtf/mtf_entities.md]: Repository --> repository

File: guides/v2.0/mtf/mtf_entities/mtf_fixture-repo.md [hunk 21]
```diff
@@ -305,9 +305,9 @@ The repository code for these cases follows:
 
 ## Configuration repository {#mtf_repository_config}
 
-The configuration repository stores predefined data sets for Magento configuration settings. It is the repository for the <a href="{{ site.mage2000url }}app/code/Magento/Config">Magento_Config module</a> and is named `ConfigData.xml`.
+The configuration repository stores predefined data sets for Magento configuration settings. It is the repository for the [Magento\_Config module]({{ site.mage2000url }}app/code/Magento/Config) and is named `ConfigData.xml`.
```
Comment (shrielenee) [inline guides/v2.0/mtf/mtf_entities/mtf_fixture-repo.md]: Should this  file path in the URL be in code formatting?

File: guides/v2.0/mtf/mtf_entities/mtf_fixture-repo.md [hunk 22]
```diff
@@ -305,9 +305,9 @@ The repository code for these cases follows:
...
 
-Let's see the following example of configuration settings for <a href="{{ site.mage2000url }}app/code/Magento/Authorizenet">Authorize.Net module</a> `<magento2_root_dir>/dev/tests/functional/tests/app/Magento/Authorizenet/Test/Repository/ConfigData.xml`.
+Let's see the following example of configuration settings for [Authorize.Net module]({{ site.mage2000url }}app/code/Magento/Authorizenet) `<magento2_root_dir>/dev/tests/functional/tests/app/Magento/Authorizenet/Test/Repository/ConfigData.xml`.
```
Comment (shrielenee) [inline guides/v2.0/mtf/mtf_entities/mtf_fixture-repo.md]: "Authorize.net"

File: guides/v2.0/mtf/mtf_entities/mtf_handler.md [hunk 23]
```diff
@@ -266,9 +266,9 @@ class Curl extends AbstractCurl
 
 ### cURL authentication classes {#mtf_handler_decor}
 
-In the previously mentioned example of the <a href="#mtf_curl_script">Curl.php</a> code, authentication in the Admin is realized using the `BackendDecorator` class.
+In the previously mentioned example of the [Curl.php](#mtf_curl_script) code, authentication in the Admin is realized using the `BackendDecorator` class.
```
Comment (shrielenee) [inline guides/v2.0/mtf/mtf_entities/mtf_handler.md]: Should this "Curl.php" be in code formatting?

File: guides/v2.0/mtf/mtf_update.md [hunk 24]
```diff
@@ -24,11 +24,11 @@ Step 1.    To avoid conflicts with the previous version, remove directory `<mage
 
 Step 2.    Remove file `<magento2_root_dir>/dev/tests/functional/composer.lock`.
 
-<div class="bs-callout bs-callout-info" id="info">
+<div class="bs-callout bs-callout-info" markdown="1">
   <p><b>Why:</b> {% glossarytooltip d85e2d0a-221f-4d03-aa43-0cda9f50809e %}Composer{% endglossarytooltip %} reads dependencies from <code>composer.lock</code> instead of reading <code>composer.json</code>. File <code>composer.lock</code> currently is not maintained.</p>
 </div>
 
-Step 3.    <a href="{{ page.baseurl }}/mtf/mtf_installation.html#mtf_install_perform">Perform and check installation.</a>
+Step 3.    [Perform and check installation.]({{ page.baseurl }}/mtf/mtf_installation.html#mtf_install_perform)
```
Comment (shrielenee) [inline guides/v2.0/mtf/mtf_update.md]: The period should be outside the linked text.

File: guides/v2.0/release-notes/ReleaseNotes2.0.10CE.md [hunk 25]
```diff
@@ -12,25 +12,25 @@ We are pleased to present Magento Open Source (formerly Community Edition) 2.0.1
...
 
 ### Why are we adding new APIs in a patch release?
 {:.no_toc}
 
 
-<i>These new interfaces will not break any existing customizations or extensions</i>.   See <a href="https://alankent.me/category/magento/" target="_blank">Alan Kent’s blog about Magento</a> for more information about these features and Magento’s use of semantic versioning.
+<i>These new interfaces will not break any existing customizations or extensions</i>.   See [Alan Kent’s blog about Magento](https://alankent.me/category/magento/){: target="_blank"} for more information about these features and Magento’s use of semantic versioning.
```
Comment (shrielenee) [inline guides/v2.0/release-notes/ReleaseNotes2.0.10CE.md]: Too many spaces before "See Alan Kent's..."

File: guides/v2.0/release-notes/ReleaseNotes2.0.10EE.md [hunk 26]
```diff
@@ -12,24 +12,24 @@ We are pleased to present Magento Commerce (formerly Enterprise Edition) 2.0.10.
...
 
 #### Why are we adding new APIs in a patch release?
 {:.no_toc}
 
-<i>These new interfaces will not break any existing customizations or extensions.</i>   See <a href="https://alankent.me/category/magento/" target="_blank">Alan Kent’s blog about Magento</a> for more information about these features and Magento’s use of semantic versioning.
+<i>These new interfaces will not break any existing customizations or extensions.</i>   See [Alan Kent’s blog about Magento](https://alankent.me/category/magento/){: target="_blank"} for more information about these features and Magento’s use of semantic versioning.
```
Comment (shrielenee) [inline guides/v2.0/release-notes/ReleaseNotes2.0.10EE.md]: Too many spaces before "See..." sentence.

File: guides/v2.0/ui-components/ui-secondary-inline.md [hunk 27]
```diff
@@ -18,7 +18,7 @@ The Listing component can use the Inline Edit if it is configured and enabled.
 
 #### Implementation of Inline Edit
```
Comment (shrielenee) [inline guides/v2.0/ui-components/ui-secondary-inline.md]: Should this file path be in code formatting?

**Pull request closed without merging**